lib/utils.pm

```diff
@@ -77,6 +78,7 @@ our @EXPORT = qw(
   reconnect_s390
   set_hostname
   wait_supportserver
+  pause_until unpause
```

Comment:
I think it would be better when this locking stuff would stay in locking
Like this we can put everything into utils or create allias or wrapper in utils for everything
utils eq chuck_norris ? ^^

Reply:
I moved this to lockapi.pm, let's wait for tools team opinion

Reply:
I will just drop last commit ;)